### PR TITLE
[X-0] Set up database and liquibase

### PR DESCRIPTION
**Pull request by fabioir**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,26 +12,27 @@
 	<artifactId>xantar-core</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
 	<name>xantar-core</name>
-	<description>Tool for planning meals</description>
+	<description>Demo project for Spring Boot</description>
```
Comment (fabioir) [inline pom.xml]: Can we rework this?

File: pom.xml [hunk 2]
```diff
@@ -12,26 +12,27 @@
...
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-webflux</artifactId>
-		</dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>4.4.3</version>
+        </dependency>
```
Comment (fabioir) [inline pom.xml]: This indentation looks a little messed up to me 😂

File: src/main/resources/db/changelog/db.changelog-1.0-initial_structure.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    
+	<changeSet author="guerreal" id="create meals table">
+	
+
```
Comment (fabioir) [inline src/main/resources/db/changelog/db.changelog-1.0-initial_structure.xml]:
```suggestion
```

File: src/main/resources/db/changelog/db.changelog-1.0-initial_structure.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		<addForeignKeyConstraint baseTableSchemaName="xantar" baseTableName="slots" baseColumnNames="slot_id" constraintName="FK_SLOT_ID_SLOT_SLOT_ID" referencedTableSchemaName="xantar" referencedTableName="master_slots" referencedColumnNames="id" />
+
+        <rollback>
+			<dropForeignKeyConstraint baseTableName="slots" baseTableSchemaName="xantar"/>
+            <dropTable tableName="master_slots" schemaName="xantar"/>
+            <dropTable tableName="slots" schemaName="xantar"/>
+        </rollback>
+    </changeSet>
+
+</databaseChangeLog>
```
Comment (fabioir) [inline src/main/resources/db/changelog/db.changelog-1.0-initial_structure.xml:54]: Please check file formatting (indentation, line breaks)

**Review by fabioir — COMMENTED** [review]
Once you've had a look at the comments it will LGTM 👏

**Review by algpuentes — APPROVED** [review]

**Pull request merged**